src/styles.css

```
@@ -199,6 +199,24 @@
 	}
 	body {
 		@apply bg-background text-foreground;
+		padding-top: env(safe-area-inset-top);
```

Stylelint (17.12.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)